codespell_lib/data/dictionary.txt

```diff
@@ -18951,6 +18955,7 @@ practicly->practically
 practicval->practical
 practioner->practitioner
 practioners->practitioners
+practise->practice
```

Comment:
Not a typo:
https://en.wikipedia.org/wiki/Practice_(learning_method)

Possibly could go in the BE to AmE dictionary:
> In English, practice is the noun and practise is the verb, but in American-English, the historically erroneous confusion of cases in America means it is now common for practice to be used both as a noun and a verb